requirements/edx/base.txt

```diff
@@ -146,7 +146,7 @@ bok-choy==0.5.3
 chrono==1.0.2
 coverage==4.0.2
 ddt==0.8.0
-diff-cover==0.9.0
+git+git://github.com/bjacobel/diff-cover.git@bjacobel/eslint#egg=diff-cover
```

Comment:
Do we have a plan for getting this merged upstream? Who will own the followup?

This should be moved from base.txt to github.txt and follow the pattern there. 
That is, git+https://github.com/... 
Also I think there was some trick to adding a version number so that environments that go back and forth between branches (e.g. devstack) keep proper track of which version of a package is installed in the venv. But I don't myself know what that trick is. I'm pretty sure that @nedbat does.